vita3k/shader/src/spirv_recompiler.cpp

```diff
@@ -699,7 +699,7 @@ static SpirvShaderParameters create_parameters(spv::Builder &b, const SceGxmProg
 
     spv::Id f32_v4_type = b.makeVectorType(f32_type, 4);
     spv::Id pa_arr_type = b.makeArrayType(f32_v4_type, b.makeIntConstant(32), 0);
-    spv::Id sa_arr_type = b.makeArrayType(f32_v4_type, b.makeIntConstant(32), 0);
+    spv::Id sa_arr_type = b.makeArrayType(f32_v4_type, b.makeIntConstant(40), 0);
```

Comment:
This is a sad workaround. No clue how we get sa indices higher than 128, probably smart to look into.